geode-core/src/test/java/org/apache/geode/distributed/LocatorLauncherTest.java

```diff
@@ -30,7 +32,7 @@
 public class LocatorLauncherTest {
 
   @Test
-  public void canBeMocked() throws Exception {
+  public void canBeMocked() {
```

Comment:
I think you should revert the changes to this file. Nothing here is pertinent to the PR.